tests/test_doctestplus.py

```diff
@@ -1142,7 +1142,7 @@ def f():
 
 
 @pytest.mark.xfail(
-        python_version() == '3.11.9',
+        python_version() in {'3.11.9', '3.12.3'},
```

Comment:
Unlikely we will introduce duplicate entries here, so `set` isn't strictly necessary?

```suggestion
        python_version() in ('3.11.9', '3.12.3'),
```

Thanks!